src/main/java/com/google/graphgeckos/dashboard/components/BuildBotData.java

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.components;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.cloud.Timestamp;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.data.annotation.Transient;
+import org.springframework.lang.NonNull;
+
+/**
+ * Contains the information retrieved from a single build bot. It is used as a member of BuildInfo...
+ * of {@link BuildInfo}
+ */
+@Entity(name = "builder")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BuildBotData {
+
+  @Transient
```

Comment:
Why is this transient?

Reply:
Because we don't need to store it in the database.

Reply:
Why do we need this to be a member of this class then?